README.adoc

```diff
@@ -128,6 +128,30 @@ The url where the registration-service is running.
 export REGISTRATION_SERVICE_URL=registration-service-url
 ----
 
+=== CONSOLE_NAMESPACE
+The console route namespace. If CONSOLE_NAMESPACE env var is not provided, it will default to "openshift-console"
+
+[source]
+----
+export CONSOLE_NAMESPACE=console-namespace
+----
+
+=== CHE_NAMESPACE
+The che route namespace. If CHE_NAMESPACE env var is not provided, it will default to "toolchain-che"
+
+[source]
+----
+export CHE_NAMESPACE=che-namespace
+----
+
+=== CHE_NAME
```

Comment:
`CHE_NAME` is a bit ambiguous. Let's use `CHE_ROUTE_NAME` instead.

Reply:
good idea! f7005505adaa796527beb28f7c5532d06d47f8fe